packages/grid/x-data-grid-pro/src/tests/printExport.DataGrid.test.tsx

```diff
@@ -54,6 +56,13 @@ describe('<DataGridPro /> - Print export', () => {
     },
   ];
 
+  before(function beforeHook() {
+    if (isJSDOM) {
+      // Can't access the document.body to print
+      this.skip();
```

Comment:
If adding an option to pass the URL takes too much time I don't mind just duplicating `createDOM.js`.

Reply:
I have tried https://github.com/mui/mui-x/pull/6273 to solve the dependency on the jsdom configuration.

I have opened https://github.com/mui/material-ui/pull/34449 because I suspect that the logic was added to fix a symptom but not the root problem (see the linked PR inside it for what the root was).

Reply:
> I don't mind just duplicating createDOM.js.

@m4theushw It's already duplicated. In https://github.com/mui/mui-x/pull/5714/files#diff-5915ab32081ceded46f473b03d334fbf63caf0633659b742eff53bab92ddf203R19, I think that we saw one of the costs we pay for it. Is it the best option? I don't know, but I personally struggle to envision fundamental reasons for having different JSDOM environments.

Reply:
If we can have both a single `createDOM` AND the print tests working that's of course better :ok_hand:

Reply:
Seems to work :ok_hand: